src/Features/Lsif/Generator/Microsoft.CodeAnalysis.LanguageServerIndexFormat.Generator.csproj

```diff
@@ -65,6 +65,9 @@
     <PackageReference Include="System.Memory" Version="$(SystemMemoryVersion)" />
   </ItemGroup>
 
+  <!-- Since we may be loading analyzers in-process, we need to have the appropriate binding redirects in place -->
```

Comment:
Mind sharing a bit more on what this does?

Reply:
So MSBuild has support for generating binding redirects, and generates them based on what reference it sees that needs it: if you are referencing A.dll and B.dll, and B.dll has a different version of A than you're using, it'll add a reference between those. This is an extra set of targets we have in Roslyn which forces binding redirects from 0 up to the deployed version for all compiler binaries, since we need those to be there for back compat. This is also used for csc, vbc, etc.

Reply:
ooooo I see I see. Thank you!